src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartDetailAdminSearchPanel.cshtml

```diff
@@ -0,0 +1,37 @@
+@using System.Globalization
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@using OrchardCore.Lists.ViewModels
+@using OrchardCore.Localization
+@model OrchardCore.Lists.ViewModels.ListPartViewModel
+@{
+    var dropdownClassList = "dropdown-menu";
+    if (CultureInfo.CurrentUICulture.IsRightToLeft())
+    {
+        dropdownClassList += " dropdown-menu-start";
+    }
```

Comment:
```suggestion
    var dropdownClassList = CultureInfo.CurrentUICulture.IsRightToLeft() 
       ? "dropdown-menu dropdown-menu-start" 
        : "dropdown-menu";
```

Reply:
Next Harvest, I am nominating you to the  formatting-bot award :)

Reply:
lol